eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -44,6 +44,9 @@ parameters:
 - name: MatrixReplace
   type: object
   default: []
+- name: SkipAggregateReports
```

Comment:
Where is this used?

Reply:
Looking at https://github.com/Azure/azure-sdk-for-java/search?q=SkipAggregateReports&type= perhaps you missed this in your last PR. However shouldn't this still be passed down somewhere?

Reply:
Yeah, I had missed passing this param down to the test yml. Fixed now.